src/base/init/init.c

```diff
@@ -344,6 +344,13 @@ void low_mem_init(void)
     leavedos(98);
     return;
   }
+  if (config.xms_size)
+    config.xms_map_size = (mem_16M - (memsize + EXTMEM_SIZE)) & PAGE_MASK;
+  phys_rsv = EXTMEM_SIZE + config.xms_map_size;
+  /* High memory needs to be mapped for EXTMEM/XMS/VGAEMU independent
+     of config.dpmi and config.dpmi_lin_rsv_size values */
+  dpmi_size = _max(phys_rsv + config.vgaemu_memsize * 1024, dpmi_size);
```

Comment:
Please use some other name, as
I guess this is not a dpmi_size now,
but some size to reserve.